set.mm

```diff
@@ -419171,116 +419164,181 @@ the theorem forms ("i", "d" or "g") should be the first of multiple
+       except currently for section comments, by the Metamath program
+       "MM> WRITE SOURCE source set.mm / REWRAP" command).  The file ends with
+       an empty line.  There are no trailing spaces.
+       </p>
        </li>
 
-       <li><b>Usage of Parentheticals.</b>
-       Usually, the comment of a theorem should contain at most one of the
-       "Revised by" and "Proof shortened by" parentheticals, see Metamath Book,
-       pp. 142-143 (there must always be a "Contributed by" parenthetical for
-       every theorem).  Exceptions for "Proof shortened by" parentheticals
-       are essential additional shortenings by a different person.  If a proof
-       is shortened by the same person, the date within the "Proof shortened
-       by" parenthetical should be updated only.  This also holds for "Revised
-       by" parentheticals, except that also more than one of such
-       parentheticals for the same person are acceptable (if there are good
-       reasons for this).
+       <li><b>Language and spelling.</b>
+       <p>
+       The MPE uses American English, e.g., we write "neighborhood" instead of
+       the British English "neighbourhood".  An exception is the word "analog",
+       which can be either a noun or an adjective (furthermore, "analog" has
+       the confounding meaning "not digital"); therefore, "analogue" is used
+       for the noun and "analogous" for the adjective.
+       </p>
+       <p>
+       Since comments may contain many space-separated symbols, we use the
+       older convention of two spaces after a period ending a sentence, to
+       better separate sentences (this is also achieved by the Metamath program
+       "MM> WRITE SOURCE source set.mm / REWRAP" command).
+       <p>
+       <p>
+       When compound words have several variants, we prefer the concatenated
+       variant (e.g., nonempty, nontrivial, nonpositive, nonzero,
+       nonincreasing...).
+       </p>
        </li>
 
-       <li><b>Input format.</b>
-       The input is in ASCII with two-space indents.  Tab characters are not
-       allowed.  Use embedded math comments or HTML entities for non-ASCII
-       characters (e.g., "&amp;eacute;" for "&eacute;").
+       <li><b>Quotation style.</b>
+       <p>
+       We use the "logical quotation style", which means that when a quoted
+       text is followed by punctuation not pertaining to the quote, then the
+       quotation mark precedes the punctuation (like at the beginning of this
+       sentence).  We use the double quote as default quotation mark (since the
+       single quote also serves as apostrophe), and the single quote in the
+       case of a nested quotation.
+       </p>
        </li>
 
-       <li><b>Comments and layout.</b>
+       <li><b>Sectioning and section headers.</b>
+       <p>
+       The database set.mm has a sectioning system with four levels of titles,
+       signaled by "decoration lines" which are 79-character long repetitions
```

Comment:
Maybe 80 characters, too.  This would also be a better match for separations using alternative characters =-=-.